Lib/test/test_sqlite3/__main__.py

```diff
@@ -0,0 +1,4 @@
+from test.test_sqlite3 import load_tests
+import unittest
+
+unittest.main()
```

Comment:
While you wouldn't expect this module to be directly imported unless it's being executed, you just never know. 😄

```suggestion
if __name__ == "__main__":
    unittest.main()
```

Reply:
Agreed :) Thanks